src/illus/norockets.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="180" height="124" viewBox="0 0 180 124" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Since the illustration only contains the rocket, more suitable name will be `rocket.svg`